api/conftest.py

```diff
@@ -63,6 +67,51 @@
 )
 from users.models import FFAdminUser, UserPermissionGroup
 
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
+        "--ci",
+        action="store_true",
+        default=False,
+        help="Enable CI mode",
+    )
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_configure(config: pytest.Config) -> None:
+    if (
+        config.option.ci
+        and config.option.dist != "no"
```

Comment:
I'm surprised that the option's value comes out as `"no"` instead of `False`.

Reply:
`"no"` is, in fact, one of xdist's distribution modes. See [here](https://pytest-xdist.readthedocs.io/en/latest/distribution.html#running-tests-across-multiple-cpus).